tensorflow_datasets/scripts/documentation/templates/dataset_doc_template.py

```diff
@@ -0,0 +1,382 @@
+"""Dataset catalog documentation template.
```

Comment:
I think this should be moved out of `template`, as template was the folder for the mako files.